### PR TITLE
go-junit-report: init at 2018-06-14 (385fac0ce9a)

### PR DESCRIPTION
**Pull request by cryptix**
###### Motivation for this change

tl;dr: pipe `go test` results into this and get enkins reports.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by cryptix** [comment]
@GrahamcOfBorg build go-junit-report

**Comment by cryptix** [comment]
squashed and rebased on master.

used this before nix and now over the weekend on a nix from jenkins witht his in its `packages`.

File: pkgs/development/tools/go-junit-report/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    inherit rev;
+    owner = "jstemmer";
+    repo = "go-junit-report";
+    sha256 = "109zs8wpdmc2ijc2khyqija8imay88ka6v50xvrpnnwnd3ywckxi";
+  };
+
+  # TODO: goDeps = ./deps.nix;
+
+  meta = with stdenv.lib; {
+    description = "Converts go test output to an xml report, suitable for applications that expect junit xml reports (e.g. Jenkins).";
```
Comment (c0bw3b) [inline pkgs/development/tools/go-junit-report/default.nix]: Can you remove the period at the end of the description please?
(as per [nixpkgs standards](https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes))

**Comment by c0bw3b** [comment]
@GrahamcOfBorg build go-junit-report

**Comment by cryptix** [comment]
Will update the description. While I'm at it: what about the version/date in the commit title? The package doesn't have a clear version published so I chose the date of the latest commit.

**Comment by cryptix** [comment]
also removed the `TODO: godeps.nix` - this code is self-contained (expect stdlib).

**Comment by c0bw3b** [comment]
Your commit message and package naming is good as it is.
You followed the [manual recommendations](https://nixos.org/nixpkgs/manual/#sec-package-naming) for unstable packages where we fetch a given commit and not a stable revision (i.e. _unstable_ in the name and date of the fetched commit as version)

**Comment by cryptix** [comment]
perfect - just pushed the cleaned up commit and rebased it on master. thanks for the review!!

**Review by c0bw3b — APPROVED** [review]
nox-review ok
```shell
lrwxrwxrwx 1 xxxxx users 83 27 oct.  15:52 result-bin -> /nix/store/l91v09dgdlydq87avvkmvlip62g33281-go-junit-report-unstable-2018-06-14-bin
```

Thanks for the contribution
